web/src/beta/lib/core/engines/Cesium/core/Imagery.tsx

```diff
@@ -45,13 +44,13 @@ export default function ImageryLayers({ tiles, cesiumIonAccessToken }: Props) {
     <>
       {tiles
         ?.map(({ id, ...tile }) => ({ ...tile, id, provider: providers[id]?.[2] }))
-        .map(({ id, tile_opacity: opacity, tile_minLevel: min, tile_maxLevel: max, provider }, i) =>
+        .map(({ id, tile_opacity: opacity, tile_zoomLevel, provider }, i) =>
           provider ? (
             <ImageryLayer
               key={`${id}_${i}_${counter.current}`}
               imageryProvider={provider}
-              minimumTerrainLevel={min}
-              maximumTerrainLevel={max}
+              minimumTerrainLevel={tile_zoomLevel?.[0] ?? 0}
+              maximumTerrainLevel={tile_zoomLevel?.[1] ?? 30}
```

Comment:
We aren't handling any other value's fallbacks at this level. can you set the. `?? 0` `?? 30` where we are processing tile_zoomLevel? 🙏🏻

Reply:
I think there is no need for process it  as it will be set by the defaultValue, I will just remove it